test/lit.cfg

```diff
@@ -129,6 +132,43 @@ if config.test_exec_root is None:
 
 ###
 
+class SwiftTest(lit.formats.ShTest):
+    def __init__(self, coverage_mode=None, execute_external=True):
```

Comment:
It comes from `lit.site.cfg.in` -- CMake fills it in with the appropriate value, and it's used to determine the pre- and post-test behavior